### ISSUE TITLE
Add plugin to init script

### ISSUE THREAD

**Issue by huehnerlady** [autonomousapps/dependency-analysis-android-gradle-plugin#240]
**Plugin version**
0.55.0

**Gradle version**
6.5.1

Hi,
I would like to use this plugin but do not check it in as I wanna try it first before I propose it to my colleagues.
So I would like to add this plugin to my init.gradle.kts

My init.gradle.kts file:

```
import com.autonomousapps.DependencyAnalysisPlugin

initscript {
    repositories {
         mavenCentral()
    }

    dependencies {
        classpath("com.autonomousapps:dependency-analysis-gradle-plugin:0.55.0")
    }
}
rootProject {
    apply<DependencyAnalysisPlugin>()
}
```

I try to run `./gradlew buildHealth`, but I get the following error:

```
* What went wrong:
A problem occurred configuring root project 'my-project'.
> Type org.jetbrains.kotlin.gradle.dsl.KotlinProjectExtension not present

```

Do you have any idea what I can do to fix that?

**Comment by autonomousapps** [comment]
It's a classloader issue. The plugin analyzes Kotlin code, and so has a `compileOnly` dependency on the Kotlin plugin. It will only hit code paths that try to access Kotlin plugin-supplied symbols if the Kotlin plugin is actually applied. However, in your case this plugin will be loaded in the parent classloader of the classloader that has the Kotlin plugin. Parent classloaders don't see the classes of child classloaders (only the inverse).

**tl;dr** the solution is to add the Kotlin plugin as a `classpath` dependency in your snippet above. At least I'm pretty sure of this.
```
classpath("org.jetbrains.kotlin:kotlin-gradle-plugin")
```

**Comment by huehnerlady** [comment]
I cannot get it to work. But I did not know that it is purely for kotlin code, I must have missed that. As we have a lot of groovy projects, I guess this is not the right plugin for me, sorry to bother you :)

**Comment by autonomousapps** [comment]
It's not purely for Kotlin code. It's for Kotlin and Java. It does not presently do any analysis of Groovy code.

When I said "the plugin analyzes Kotlin code", what I meant was that one thing it does is
```kotlin
pluginManager.withPlugin("kotlin-plugin") { // coding in github
  // it only deals with Kotlin code in here
}
```
Your error made me think something was happening when it reacted to the presence of the Kotlin plugin.

**Comment by huehnerlady** [comment]
I see, sorry for the missunderstanding. but as it does not support groovy I guess my problem still exists. but I will keep this in mind for future, we currently migrate a bit towards kotlin :)
